test/unittest/dds/xtypes/serializers/idl/CMakeLists.txt

```diff
@@ -0,0 +1,92 @@
+# Copyright 2024 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+configure_file(
+    ${CMAKE_CURRENT_SOURCE_DIR}/types/alias_struct/alias_struct.idl
+    ${CMAKE_CURRENT_BINARY_DIR}/types/alias_struct/alias_struct.idl
+    COPYONLY)
+configure_file(
+    ${CMAKE_CURRENT_SOURCE_DIR}/types/array_struct/array_struct.idl
+    ${CMAKE_CURRENT_BINARY_DIR}/types/array_struct/array_struct.idl
+    COPYONLY)
+configure_file(
+    ${CMAKE_CURRENT_SOURCE_DIR}/types/bitmask_struct/bitmask_struct.idl
+    ${CMAKE_CURRENT_BINARY_DIR}/types/bitmask_struct/bitmask_struct.idl
+    COPYONLY)
+configure_file(
+    ${CMAKE_CURRENT_SOURCE_DIR}/types/bitset_struct/bitset_struct.idl
+    ${CMAKE_CURRENT_BINARY_DIR}/types/bitset_struct/bitset_struct.idl
+    COPYONLY)
+configure_file(
+    ${CMAKE_CURRENT_SOURCE_DIR}/types/enum_struct/enum_struct.idl
+    ${CMAKE_CURRENT_BINARY_DIR}/types/enum_struct/enum_struct.idl
+    COPYONLY)
+configure_file(
+    ${CMAKE_CURRENT_SOURCE_DIR}/types/extensibility_struct/extensibility_struct.idl
+    ${CMAKE_CURRENT_BINARY_DIR}/types/extensibility_struct/extensibility_struct.idl
+    COPYONLY)
+configure_file(
+    ${CMAKE_CURRENT_SOURCE_DIR}/types/key_struct/key_struct.idl
+    ${CMAKE_CURRENT_BINARY_DIR}/types/key_struct/key_struct.idl
+    COPYONLY)
+configure_file(
+    ${CMAKE_CURRENT_SOURCE_DIR}/types/map_struct/map_struct.idl
+    ${CMAKE_CURRENT_BINARY_DIR}/types/map_struct/map_struct.idl
+    COPYONLY)
+configure_file(
+    ${CMAKE_CURRENT_SOURCE_DIR}/types/primitives_struct/primitives_struct.idl
+    ${CMAKE_CURRENT_BINARY_DIR}/types/primitives_struct/primitives_struct.idl
+    COPYONLY)
+configure_file(
+    ${CMAKE_CURRENT_SOURCE_DIR}/types/sequence_struct/sequence_struct.idl
+    ${CMAKE_CURRENT_BINARY_DIR}/types/sequence_struct/sequence_struct.idl
+    COPYONLY)
+configure_file(
+    ${CMAKE_CURRENT_SOURCE_DIR}/types/string_struct/string_struct.idl
+    ${CMAKE_CURRENT_BINARY_DIR}/types/string_struct/string_struct.idl
+    COPYONLY)
+configure_file(
+    ${CMAKE_CURRENT_SOURCE_DIR}/types/struct_struct/struct_struct.idl
+    ${CMAKE_CURRENT_BINARY_DIR}/types/struct_struct/struct_struct.idl
+    COPYONLY)
+configure_file(
+    ${CMAKE_CURRENT_SOURCE_DIR}/types/union_struct/union_struct.idl
+    ${CMAKE_CURRENT_BINARY_DIR}/types/union_struct/union_struct.idl
+    COPYONLY)
```

Comment:
```suggestion
file(GLOB_RECURSE IDL_FILES ${CMAKE_CURRENT_SOURCE_DIR}/types/**/*.idl)

foreach(idl_file ${IDL_FILES})
    file(RELATIVE_PATH relative_path ${CMAKE_CURRENT_SOURCE_DIR} ${idl_file})
    get_filename_component(relative_dir ${relative_path} DIRECTORY)
    file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/${relative_dir})
    file(COPY ${idl_file} DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/${relative_dir})
endforeach()
```

Reply:
Could you please also modify `test/unittest/dds/xtypes/serializers/json/CMakeLists.txt` with the following?

```
file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/types/comprehensive_type/json
    DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/types/comprehensive_type
    )
```